### DISCUSSION TITLE
q-table: default all rows selected

### DISCUSSION THREAD

**Discussion by marlonguita** [quasarframework/quasar#14190] [General - Components / Directives / etc] [answered]
Hi there!
How can I get all rows seleted when my component is shown? I don't wanna select row by row. By default, all rows should be selected. How can I do this?

Tks!!!

**Comment by MilosPaunovic** [comment, marked as answer]
Set the array of selected rows to be the same as the array containting all row data.
https://codepen.io/milospaunovic/pen/QWmZvPV?editors=1010

**Reply by marlonguita** [reply]
Thanks, @MilosPaunovic !
